include/vigra/mathutil.hxx

```diff
@@ -164,6 +164,17 @@ VIGRA_DEFINE_MISSING_ABS(signed long long)
 using std::isinf;
 using std::isnan;
 
+#ifdef VIGRA_NO_INT_ISNAN
```

Comment:
@hmeine [reported][1] that this extra definition of `isnan` causes errors if the std library already provides its own.  That's why I added this condition.

[1]: https://github.com/ukoethe/vigra/pull/225#issuecomment-69341499